chordspy/app.py

```
@@ -112,26 +119,27 @@ def launch_application():
-    app_name = data.get('app')
+    module_name = data.get('app')
     
-    if not app_name:
+    if not module_name:
         return jsonify({'status': 'error', 'message': 'No application specified'}), 400
     
     # Check if app is already running
-    if app_name in running_apps and running_apps[app_name].poll() is None:
-        return jsonify({'status': 'error', 'message': f'{app_name} is already running','code': 'ALREADY_RUNNING'}), 400
+    if module_name in running_apps and running_apps[module_name].poll() is None:
+        return jsonify({'status': 'error', 'message': f'{module_name} is already running','code': 'ALREADY_RUNNING'}), 400
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)